### DISCUSSION TITLE
Get coordinates from tables texts.

### DISCUSSION THREAD

**Discussion by molhineko** [jsvine/pdfplumber#1180] [Ask for help with specific PDFs]
`page.extract_tables()` is able to extract text from each cell preciously. I want to get text coordinates from specific cell text. So, I am using `page.search(text)` to get coordinates. However, `page.search()` seems using `page.extract_text()` to search text. `page.extract_text()` is not works well for table content. It is sometimes sentence sequences are broken by other cell's text. So, I want to get coordinates along with `page.extract_tables()`. Are there such option on it ? Does anyone have idea ?

**Comment by cmdlineluser** [comment]
There is also `.find_tables()` which gives you the actual Table objects.

You can then access `table.rows` / `row.cells` / etc.

**Reply by jsvine** [reply]
Thank you @cmdlineluser, and seconding your suggestion.
